engine/classes/Elgg/Di/ServiceProvider.php

```diff
@@ -257,8 +264,21 @@ public function __construct(\Elgg\Config $config) {
 
 		$this->setClassName('usersTable', \Elgg\Database\UsersTable::class);
 
+		$this->setFactory('viewPaths', function(ServiceProvider $c) {
+			return new \Elgg\Views\DirectoryPathRegistry($c->elggRootDir->chroot('views'), $c->viewtypes);
```

Comment:
`->chroot()` just applies a filter after deep scanning. I can't imagine ever wanting that behavior.